README.md

```diff
@@ -1,127 +1,31 @@
-Overview
-========
+Astronomer's Airflow + Astro workshop
+=====================================
 
-This workshop is designed to give users an intro to Astro, Astronomer's flagship cloud product, as well as familiarize users with the latest features available in Airflow.
+This repo contains workshops designed to give users an intro to Astro, Astronomer's flagship cloud product, as well as familiarize users with the latest features available in Airflow. 
 
-## Getting started
+This branch contains information for workshop hosts and contributors.
 
-Ensure you have v1.19 of the Astronomer CLI installed. To complete the exercises, start Airflow on your local machine by running `astro dev start`. 
+## Instructions and prerequisites for all workshops
 
-Refer to the instructions below for each exercise. All DAGs can run locally without connecting to external systems.
+All workshops have their own branch in this repo (see [Choosing a workshop](#choosing-a-workshop) for more info about the different branched). The `README` in each branch contains instructions for the exercises. The file structure is the same as an Astro project, with an additional folder for `exercise-solutions` where relevant.
 
-Possible solutions can be found in the `solutions` folder, although for some exercises there are multiple ways to implement.
+Workshops are intended to be given in person or virtually, with one Astronomer presenter and at least one other Astronomer helper. Workshop content is not designed for participants to complete asynchronously. 
 
+For each workshop, participants will need:
 
-## Part 1: Creating Deployments
-In this section you will create multiple environments on Astro, from the Cloud UI and the API.
+- An Astro account (a free trial will work, but ensure it has access to the features covered in the workshop before beginning).
+- The Astro CLI
+- Docker 4.0 or greater
 
-Feel free to use the following resources:
 
-- [Create a deployment](https://docs.astronomer.io/astro/create-deployment)
-- [Manage Deployments programmatically using deployment files](https://docs.astronomer.io/astro/manage-deployments-as-code#create-a-deployment-from-a-template-file)
+## Choosing a workshop
 
-### Exercise 1: Create a deployment from the Astro Cloud UI
+There are multiple variations of this workshop available on different branches in the repository:
```

Comment:
```suggestion
There are multiple workshops available on different branches in the repository:
```

Very nitty nit: Are they branches multiple variations of the same workshop or multiple workshops in the repo? 😅 I understood the rest of the document as being the latter.